simple-facebook-instant-articles.php

```diff
@@ -240,6 +231,17 @@ public function image_shortcode( $atts, $content = '' ) {
 	 * can be rendered into FB IA format.
 	 */
 	public function render_post_content() {
+
+		// Shortcodes - overwrite WP native ones with FB IA format.
+		add_shortcode( 'gallery', array( $this, 'gallery_shortcode' ) );
+		add_shortcode( 'caption', array( $this, 'image_shortcode' ) );
+
+		// Render social embeds into FB IA format.
+		add_filter( 'embed_handler_html', array( $this, 'fb_formatted_social_embeds' ), 10, 3 );
+		add_filter( 'embed_oembed_html', array( $this, 'fb_formatted_social_embeds' ), 10, 4 );
+
+		// Render post content via DOM - to format it into FB IA format.
+		// DO it last, so content was altered via WP native hooks as much as possible.
```

Comment:
You commented `DO it last` - but you're not hooking the filter in late. If you need this - I would hook in on `the_content` priority 100 or something. As the above code is affecting different filters - I don't think the order here makes any difference!